client/packages/reports/src/DetailView/DetailView.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { NothingHere, useBreadcrumbs } from '@openmsupply-client/common';
+import { useDisplayReport, useReportStore } from '@openmsupply-client/system';
+import { Environment } from '@openmsupply-client/config';
+
+export const DetailView = () => {
+  const { report, args } = useReportStore();
```

Comment:
I think this would mean regardless of what report page you go to, it would render current report ?

I think we would need to get report id from parameters, then get the reports with graphql, then user report id to get last args stored in useReportStore

useReportStore should save args by report id

Reply:
The id would need to be saved in state too since it can't be null or undefined when getting the url 👀